nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -2077,7 +2082,7 @@ class SyncAskResponse(BaseModel):
     debug: Optional[dict[str, Any]] = Field(
         default=None,
         title="Debug information",
-        description="Debug information about predict",
+        description="Debug information about the ask operation. This is only available if the `debug` parameter is set to true in the request.",  # noqa: E501
```

Comment:
Maybe we should add some notice here and in the `metrics` field that this information is subject to change at any point? I don't want people to start relying it and complain if we change the format of it (I don't want to do bw/c on these fields).

Reply:
agree!